starlette_admin/actions.py

```diff
@@ -11,6 +11,8 @@ def action(
     submit_btn_text: Optional[str] = _("Yes, Proceed"),
     icon_class: Optional[str] = None,
     form: Optional[str] = None,
+    form_template: Optional[str] = "forms/action_form.html",
+    form_context: Optional[dict] = None,
```

Comment:
```suggestion
    form_context: Optional[Dict] = None,
```

To support older Python versions, you should import Dict from typing or typing_extensions and use `Dict` type.